test/controllers/welcome_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class WelcomeControllerTest < ActionController::TestCase
```

Comment:
this is a Test::Unit test, not sure how it got generated... delete. Might as well delete the test/ directory altogether